core/src/main/java/com/google/cloud/sql/core/SslSocketFactory.java

```diff
@@ -469,7 +469,9 @@ private X509Certificate obtainEphemeralCertificate(
             new RuntimeException(
                 String.format(
                     "Unable to obtain ephemeral certificate for Cloud SQL Instance [%s]. "
-                        + "Make sure %s Editor or Owner role on the project.",
+                        + "Ensure %s the sql.instances.connect permission "
```

Comment:
"has" is part of the substituted string because one alternative is "you have" instead of "foo has"